README.md

```diff
@@ -1,39 +1,61 @@
 # testing-farm-as-github-action
 
-Testing Farm as GitHub Action is a GitHub Action for executing tests on the Testing Farm infrastructure.
+Testing Farm as GitHub Action is a GitHub Action for executing tests on the [Testing Farm Service](https://docs.testing-farm.io).
 
-The tests to run are to be described with a [`fmf` plan](https://tmt.readthedocs.io/en/latest/spec.html) by the user of this GitHub Action.
+The tests to run are to be described with a [`tmt` plan](https://tmt.readthedocs.io/en/latest/spec.html) by the user of this GitHub Action.
 Pull Request status is automatically updated after the tests are executed,
 if this option is enabled with the `update_pull_request_status` user-defined input variable.
 
-
 Before calling this GitHub Action, you must first clone your project,
 e.g., with the [Checkout V2](https://github.com/actions/checkout) GitHub Action.
 
-The Action uses ubuntu-20.04 and it is a [composite](https://docs.github.com/en/actions/creating-actions/about-custom-actions).
+The Action uses ubuntu-20.04 and it is a [composite action](https://docs.github.com/en/actions/creating-actions/about-custom-actions).
 It internally downloads needed binaries `curl` and `jq` for communicating with the Testing Farms API and parsing the responses.
  
-API key to the Testing Farm should be stored in your organization's secrets to successfully access its infrastructure.
+API key to the Testing Farm MUST be stored in your organization's secrets to successfully access its infrastructure.
+
+## Compatibility Notes
+
+⚠ Currently only testing with copr artifacts is supported.
 
 ## Action Inputs
 
+### Testing Farm
+
+| Input Name                   | Description                                                                                                                       | Default value                 |
+|------------------------------|-----------------------------------------------------------------------------------------------------------------------------------|-------------------------------|
+| `api_key`                    | Testing farm API key.                                                                                                             | empty, **required from user** |
+### Tmt Metadata
+
+| Input Name                   | Description                                                                                                                       | Default value                 |
+|------------------------------|-----------------------------------------------------------------------------------------------------------------------------------|-------------------------------|
+| `url`                        | An url to the repository with tmt metadata                                                                                        | empty, **required from user** |
```

Comment:
I like the idea of logical groups, but in addition, I would suggest a prefix, based on them. This way they can be easily distinguishable even without the README.md

```suggestion
| `tmt_url`                        | An url to the repository with tmt metadata                                                                                        | empty, **required from user** |
```
The same can be applied to the "test environment" group with `secrets` as `test_secrets`, `variables` as `test_variables`, etc.

Reply:
URL is too generic. Some prefix is really nice. 
@thrix Do you have already the API v2 definition?

Reply:
ack, ok, so git_url would be maybe better? :)

Reply:
And is `tmt_url`, `tmt_plan`, `tmt_ref` and `tmt_context` instead of `url`, `plan`, `ref` and `context`  inconvenient?

Reply:
@zmiklank it is not URL to tmt, it is a git repository URL same for ref, it is a git reference, others I agree

Reply:
Asking for next round of review, sorry I forget how github does not like squasing :(